### PR TITLE
spacy-model-en_core_web 3.3.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Actions:
1. Fix `python` in `host`
2. Add `setuptools` and `wheel` in `run`
3. Add `license_family`

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
Waiting for `spacy-transformers 1.1.5` https://github.com/AnacondaRecipes/spacy-transformers-feedstock/pull/2

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,31 +12,33 @@ package:
...
   - url: https://github.com/explosion/spacy-models/releases/download/{{ name }}trf-{{ version }}/{{ name }}trf-{{ version }}.tar.gz
-    sha256: f2296d93d63c228f7b4df2a29f23f1e9fe291180c7257ed3cb7221c4e4ae30e3
+    sha256: bc6195dd3a01efdbabca3c93f0eb4701ad9e78c05528d6fcdc2fe9df848bd3e1
     folder: trf
   - url: https://raw.githubusercontent.com/explosion/spaCy/master/LICENSE
-    sha256: b90775797175e7aa165a5508c8132da463b8ae029b419f4fc52450276d9f2872
+    sha256: c07800e058b1b544eb9d47dd81687582fe6830c1ddc47f57521fc93628685915
 
 build:
   noarch: python
```
Comment (mcg1969) [inline recipe/meta.yaml:30]: I wonder if it should be noarch: generic.

Comment (mcg1969) [reply recipe/meta.yaml:30]: I guess not since it is a pip install

Comment (skupr-anaconda) [reply recipe/meta.yaml:30]: I use the recipe from conda-forge https://github.com/conda-forge/spacy-model-en_core_web-feedstock/blob/main/recipe/meta.yaml
I guess it should be `noarch: python`. But it's only data models. I haven't any examples for those cases

**Comment by skupr-anaconda** [comment]
We can skip `s390x` as there are noarch python outputs

**Review by ScottBailey — CHANGES_REQUESTED** [review]
Generally looks good to me except I can't find the license file. Can you double check and correct this? 

https://github.com/AnacondaRecipes/spacy-model-en_core_web-feedstock/blob/update_3.3.0/recipe/meta.yaml#L159-L161

**Review by varlackc — COMMENTED** [review]
the package is still failing for s390x do we have support for this architecture? if not, we probably should write a comment stating that we do not give support for s390x at this point and skip that particular build.

**Comment by ScottBailey** [comment]
Skipping builds because they don't compile will put a roadblock in the way of a later automated build. For this reason, I tend to disagree with `skip: True  # [s390x] - reason: it doesn't build`.

**Comment by skupr-anaconda** [comment]
> the package is still failing for s390x do we have support for this architecture? if not, we probably should write a comment stating that we do not give support for s390x at this point and skip that particular build.

It's a noarch package with 4 outputs and currently we don't provide `spacy` with a lot of its dependencies on s390x. There is no need to `skip`

**Comment by skupr-anaconda** [comment]
> Generally looks good to me except I can't find the license file. Can you double check and correct this?
> 
> https://github.com/AnacondaRecipes/spacy-model-en_core_web-feedstock/blob/update_3.3.0/recipe/meta.yaml#L159-L161

The LICENSE is **inside** the release source files https://github.com/explosion/spacy-models/releases/tag/zh_core_web_sm-3.3.0: https://github.com/explosion/spacy-models/releases/download/zh_core_web_sm-3.3.0/zh_core_web_sm-3.3.0.tar.gz
We copy it for every output https://github.com/AnacondaRecipes/spacy-model-en_core_web-feedstock/blob/d2e71dfe50afb4f28576f14021e9c8df1c820db3/recipe/meta.yaml#L49-L51

**Review by ScottBailey — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
I've tested it locally:
```
conda create -n test_spacy -c c3i_test2 spacy-model-en_core_web_sm
conda activate test_spacy

❯ python
Python 3.10.4 (main, Mar 31 2022, 03:37:37) [Clang 12.0.0 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import spacy
>>> nlp = spacy.load("en_core_web_sm")
>>> doc = nlp(u"This is a sentence.")
>>> doc
This is a sentence.
>>> import spacy
>>> import en_core_web_sm
>>>
>>> nlp = en_core_web_sm.load()
>>> doc = nlp(u"This is a sentence.")
>>> doc
This is a sentence.
```

No issues

**Pull request merged**